api/nodes/views.py

```diff
@@ -1390,8 +1390,6 @@ def get_object(self):
             self.kwargs[node_link_lookup_url_kwarg],
             'node link'
         )
-        # May raise a permission denied
-        self.check_object_permissions(self.request, node_link)
```

Comment:
This still doesn't seem right. If you make a token with a osf.profile_read scope, can you access this endpoint or does it give a scope error? If you try to post to the registration view derived from this, does it prevent it or let it go through to the model?